frontend/src/scenes/dashboard/dashboards/Dashboards.tsx

```diff
@@ -0,0 +1,77 @@
+import { NewDashboardModal } from 'scenes/dashboard/NewDashboardModal'
+import { PageHeader } from 'lib/components/PageHeader'
+import { SceneExport } from 'scenes/sceneTypes'
+import { LemonButton } from 'lib/components/LemonButton'
+import { LemonDivider } from 'lib/components/LemonDivider'
+import { newDashboardLogic } from 'scenes/dashboard/newDashboardLogic'
+import { inAppPromptLogic } from 'lib/logic/inAppPrompt/inAppPromptLogic'
+import { LemonInput } from '@posthog/lemon-ui'
+import { DeleteDashboardModal } from 'scenes/dashboard/DeleteDashboardModal'
+import { DuplicateDashboardModal } from 'scenes/dashboard/DuplicateDashboardModal'
+import { NoDashboards } from 'scenes/dashboard/dashboards/NoDashboards'
+import { DashboardsTable } from 'scenes/dashboard/dashboards/DashboardsTable'
+
+export const scene: SceneExport = {
+    component: Dashboards,
+    logic: dashboardsLogic,
+}
+
+export function Dashboards(): JSX.Element {
+    const { dashboardsLoading } = useValues(dashboardsModel)
+    const { setSearchTerm, setCurrentTab } = useActions(dashboardsLogic)
+    const { dashboards, searchTerm, currentTab } = useValues(dashboardsLogic)
+    const { showNewDashboardModal } = useActions(newDashboardLogic)
+    const { closePrompts } = useActions(inAppPromptLogic)
+
+    return (
+        <div>
+            <NewDashboardModal />
+            <DuplicateDashboardModal />
+            <DeleteDashboardModal />
+            <PageHeader
+                title="Dashboards"
+                buttons={
+                    <LemonButton
+                        data-attr={'new-dashboard'}
+                        onClick={() => {
+                            closePrompts()
+                            showNewDashboardModal()
+                        }}
+                        type="primary"
+                    >
+                        New dashboard
+                    </LemonButton>
+                }
+            />
+            <Tabs
+                activeKey={currentTab}
+                style={{ borderColor: '#D9D9D9' }}
+                onChange={(tab) => setCurrentTab(tab as DashboardsTab)}
+            >
+                <Tabs.TabPane tab="All Dashboards" key={DashboardsTab.All} />
+                <Tabs.TabPane tab="Your Dashboards" key={DashboardsTab.Yours} />
+                <Tabs.TabPane tab="Pinned" key={DashboardsTab.Pinned} />
+                <Tabs.TabPane tab="Shared" key={DashboardsTab.Shared} />
+            </Tabs>
+            <div className="flex">
+                <LemonInput
+                    type="search"
+                    placeholder="Search for dashboards"
+                    onChange={setSearchTerm}
+                    value={searchTerm}
+                />
+                <div />
+            </div>
+            <LemonDivider className="my-4" />
+            {dashboardsLoading || dashboards.length > 0 || searchTerm || currentTab !== DashboardsTab.All ? (
+                <DashboardsTable />
```

Comment:
other than extracting <DashboardsTable/> and <NoDashboards/> (and removing their dependencies this file hasn't changed